expan/core/results.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, original_test_statistics, corrected_test_statistics):
 class StatisticalTestResult(JsonSerializable):
     """ This class holds the results of a single statistical test.
     
-    :param test: information about the statistical test
+    :param test: information about the statistical test (without data)
```

Comment:
I think we don't need to write this, but this is very minor :)

Reply:
removed!